### PR TITLE
Make use of plain text-enforcing qml labels

### PR DESCRIPTION
**Pull request by claucambra**
This PR:

- Replaces all use of Label and Text components in our QML code for a new EnforcedPlainTextLabel
- Adds a basic check in our CI for the use of Label components (of which there should only be one, in EnforcedPlainTextLabel)

### CODE REVIEW

File: .github/workflows/qml-label-check.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    types: [opened, synchronize, reopened]
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run label component count
+        run: |
+          MATCHESRESULT=$(grep -r -e "^\s*Label\s*[{]" . | wc -l | sed 's/ //g')
```
Comment (allexzander) [inline .github/workflows/qml-label-check.yml:16]: @claucambra Won't we have a false-positive in some other source files that are **non-.qml**, at some point, if the keyword **Label** is used elsewhere? Also, won't it be faster to run this step if we only run it for **.qml** files?

Comment (claucambra) [reply .github/workflows/qml-label-check.yml]: Good points, changed so we only check QML files

**Pull request merged**